source/components/tooltip/index.html

```diff
@@ -1,8 +1,182 @@
 ---
 title: Tooltip
+assets:
+   - helix-ui.css 
+   - helix-ui.js
 ---
 {% extends 'component.njk' %}
 {% block content %}
-<!-- explanation goes here -->
-<p>TBD</p>
+  <section id="vue-tooltipDemo">
+    <h2 class="hxSectionTitle" id="tooltip">Tooltip </h2>
+    <div class="hxRow">
+      <div class="hxCol" >
+        Position:
+        <select v-model="position">
+          <option v-for="_position in positions" :value="_position">
+            {% raw %}{{ _position.label }}{% endraw %}
+          </option>
+        </select>
+      </div>
+      <div class="hxCol">
+        <div class="demo">
+          <p>
+            Hover over icon to trigger tooltips
+            <hx-icon data-tooltip="tooltip1" type="help-circle"></hx-icon>
+          </p>
+          <hx-tooltip id="tooltip1" :position="position.value">
+            {% raw %}{{ position.label }}{% endraw %}
+          </hx-tooltip>
+          <hx-tooltip id="tooltip2" :position="position.value" trigger="click">
```

Comment:
Since this `hx-tooltip` is tied to `tooltip2` in the `div` below, it should probably be positioned there in the markup